frontends/web/src/locales/en/app.json

```diff
@@ -197,6 +197,40 @@
   "bitboxBase": {
     "connect": "Connect Base",
     "detectedBases": "Detected the following devices on your network:",
+    "errors": {
```

Comment:
this is unused in the frontend code still, right? I'd make a PR which adds both the messages as well as the frontend code, as it's easier to review it in combination.

Reply:
Correct, another reason to wait with merging this PR.
I can start adding it to the frontend code. Planing to add a bunch of the strings from the Base wizard to locize today after polishing the text yesterday.